galsim_extra/mixed_scene.py

```diff
@@ -58,8 +59,23 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         # object ends up being chosen.
         ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type']
 
+        stamp_xsize, stamp_ysize, image_pos, world_pos = super(MixedSceneBuilder, self).setup(config,base,xsize,ysize,ignore,logger)
+        # option to shear the full scene.
+        if config.get('shear_full_scene', True):            
+            shear = galsim.Shear(g1=float(base["stamp"]["shear"]["g1"]), g2=float(base["stamp"]["shear"]["g2"]))
+            S = shear.getMatrix()
+            # Find the center (tangent point) of the scene in RA, DEC. 
+            scene_center = base['wcs'].center
+            u,v = scene_center.project_rad(world_pos.ra, world_pos.dec, projection='gnomonic') # tile center units in radians
+            # shearing the position. 
+            pos = np.vstack((u, v))
+            sheared_uv = np.dot(S, pos)
```

Comment:
Easier:
```
u, v = scene_center.project(world_pos, projection='gnomonic')
pos = galsim.Position(u.rad, v.rad)
sheared_pos = pos.shear(shear)
u2 = sheared_pos.x * coord.radians
v2 = sheared_pos.y * coord.radians
world_pos = scene_center.deproject(u2, v2, projection='gnomonic')
```

Reply:
Would coord.radians work without specifying what 'coord' is?

Reply:
`import coord`

Reply:
Oh, and as Matt mentioned, the project/deproject won't work for EuclideanWCS types.  So you should guard those steps with `if wcs.iscelestial:`.  When that's not true, world_pos will already be a PositionD, so the only thing you need to do in that case is: `world_pos = world_pos.shear(shear)`.